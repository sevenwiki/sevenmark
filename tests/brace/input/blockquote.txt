{{{#quote #style="font-style:italic"
This is a quoted text block with some **bold** content.
}}}