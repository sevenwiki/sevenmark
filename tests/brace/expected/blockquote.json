[
  {
    "BlockQuoteElement": {
      "location": {
        "start": 0,
        "end": 96
      },
      "parameters": {
        "style": {
          "location": {
            "start": 9,
            "end": 37
          },
          "key": "style",
          "value": [
            {
              "Text": {
                "location": {
                  "start": 18,
                  "end": 35
                },
                "content": "font-style:italic"
              }
            }
          ]
        }
      },
      "content": [
        {
          "Text": {
            "location": {
              "start": 37,
              "end": 75
            },
            "content": "This is a quoted text block with some "
          }
        },
        {
          "Bold": {
            "location": {
              "start": 75,
              "end": 83
            },
            "content": [
              {
                "Text": {
                  "location": {
                    "start": 77,
                    "end": 81
                  },
                  "content": "bold"
                }
              }
            ]
          }
        },
        {
          "Text": {
            "location": {
              "start": 83,
              "end": 92
            },
            "content": " content."
          }
        },
        {
          "Text": {
            "location": {
              "start": 92,
              "end": 93
            },
            "content": "\n"
          }
        }
      ]
    }
  }
]